userspace/libscap/scap.c

```diff
@@ -172,6 +172,7 @@ int32_t scap_init(scap_t* handle, scap_open_args* oargs)
 	if(strcmp(engine_name, NODRIVER_ENGINE) == 0)
 	{
 		vtable = &scap_nodriver_engine;
+#if defined(__LINUX__)
```

Comment:
Isn't the `scap_linux_alloc_platform` definition dealt with on a per-platform basis at the beginning of the file already?

Reply:
Actually I think that was an old implementation that fails during initialisation. Do you think that we can remove it?